From 8ba77e3fd799d817798c6caabbe86872707d46ce Mon Sep 17 00:00:00 2001
From: "bo.yang" <bo.yang@amlogic.com>
Date: Tue, 4 Mar 2014 18:16:18 +0800
Subject: [PATCH 3590/5965] m6tvd: add vdec dtd

this will fix local video playing issue.
validated by shanwu.hu & bo.yang
---
 arch/arm/boot/dts/amlogic/meson6tvd_ref.dtd | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm/boot/dts/amlogic/meson6tvd_ref.dtd b/arch/arm/boot/dts/amlogic/meson6tvd_ref.dtd
index 99e63ff3bf0e..df8244a1f4cc 100755
--- a/arch/arm/boot/dts/amlogic/meson6tvd_ref.dtd
+++ b/arch/arm/boot/dts/amlogic/meson6tvd_ref.dtd
@@ -137,6 +137,20 @@
 		reserve-iomap = "true";
 	};
 
+/// ***************************************************************************************
+///     -       DISP&MM-A/V stream
+//$$ MODULE = "DISP&MM-A/V video dec"
+//$$ DEVICE = "vdec"
+//$$ L2 PROP_STR = "status"
+//$$ L3 PROP_U32 4 = "reg"
+	vdec {
+		compatible = "amlogic,vdec";
+		dev_name = "vdec.0";
+		status = "okay";
+		reserve-memory = <0x04000000>; // 64M
+//		reserve-iomap = "true";
+	};
+
 /// ***************************************************************************************
 ///	-	DISP&MM-PostProcess
 //$$ MODULE="DISP&MM-PostProcess"
-- 
2.19.0

